packages/shared/lib/wallet.ts

```diff
@@ -480,15 +480,15 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: NodeAuth): Promise<NodeInfo> => {
+export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
```

Comment:
This should be a "regular" `function`.

Reply:
to be moved and fixed in network info task.